geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneIndexCreationProfile.java

```diff
@@ -66,6 +75,10 @@ public String getAnalyzerClass() {
     return this.fieldAnalyzers;
   }
 
+  public String getSerializerClass() {
```

Comment:
Should this be named getSerializerClassName()?

Reply:
We had a getAnalyzerClass() method. So follow the same idea.